app/CommandExecutor.php

```diff
@@ -83,19 +102,50 @@ public function execute(string $command, string $cwd) : int
         return proc_close($proc);
     }
 
+    public function getOutputBuffer(): string
+    {
+        if (! $this->outputFileName) {
+            return '';
+        }
+
+        $output = file_get_contents($this->outputFileName);
+
+        if ($output === false) {
+            return 'Error: Unexpected failure trying to read the output file!';
```

Comment:
Let's add the output file name to this error message, so one might look at it in case of bizarre failure.